src/toolbars/examples/toolbar.js

```diff
@@ -81,10 +81,11 @@
         </pf-toolbar>
       </div>
       <div class="col-md-12" ng-if="viewType == 'listView' && showComponent">
-        <pf-list-view config="listConfig"
+        <pf-list-view
```

Comment:
nit.  not sure why config couldn't be inline with `pf-list-view`, seems like an extra line/whitespace to me.

Reply:
@dtaylor113 I guess a difference in style. I'll update it to reflect the style of the rest of the file.

Reply:
@dtaylor113 pushed config line changes